includes/Settings.php

```diff
@@ -133,8 +133,8 @@ public function add_support_links( $links, $file ) {
 	 * @return string
 	 */
 	public function user_settings_capability() {
-		$user_capability       = 'manage_woocommerce';
-		$capabilities_to_check = apply_filters( 'wpo_wcpdf_settings_user_role_capabilities', array( $user_capability ) );
+		$user_capability       = '';
```

Comment:
This will serve as the fallback if capability checks fail. I believe we should keep it as it was and instead add a filter to modify the default if needed.